data/json/mapgen/basement/basement_bionic.json

```diff
@@ -51,64 +52,64 @@
-        { "group": "homebooks", "x": 12, "y": [ 6, 8 ], "chance": 70, "repeat": [ 1, 2 ] },
-        { "group": "magazines", "x": 20, "y": [ 10, 12 ], "chance": 30 },
-        { "group": "magazines", "x": 9, "y": 7, "chance": 30 },
-        { "group": "snacks", "x": [ 17, 22 ], "y": 5, "chance": 40, "repeat": [ 1, 2 ] },
-        { "group": "snacks", "x": [ 17, 22 ], "y": 7, "chance": 40, "repeat": [ 1, 2 ] },
-        { "group": "softdrugs", "x": 22, "y": 14, "chance": 75, "repeat": [ 1, 2 ] },
-        { "group": "cleaning", "x": 11, "y": [ 2, 3 ], "chance": 70, "repeat": [ 1, 2 ] },
-        { "group": "cleaning", "x": [ 13, 14 ], "y": 1, "chance": 70, "repeat": [ 1, 2 ] },
-        { "group": "surgery", "x": [ 8, 9 ], "y": 3, "chance": 70, "repeat": [ 1, 2 ] },
-        { "group": "bionics_common", "x": 7, "y": 3 },
-        { "item": "anesthetic_kit", "x": 7, "y": 3 },
-        { "item": "television", "x": 22, "y": 10, "chance": 95 },
-        { "item": "soap", "x": 19, "y": 15, "chance": 80 },
-        { "item": "towel", "x": 20, "y": 15, "chance": 80 },
-        { "item": "towel", "x": 2, "y": 7, "chance": 60 },
-        { "item": "stereo", "x": 1, "y": 7, "chance": 50 }
+        { "group": "alcohol", "x": [ 14, 15 ], "y": 12, "chance": 96, "repeat": [ 1, 2 ] },
+        { "group": "fridgesnacks", "x": [ 14, 15 ], "y": 12, "chance": 80, "repeat": [ 1, 2 ] },
+        { "group": "homebooks", "x": [ 7, 12 ], "y": 11, "chance": 70, "repeat": [ 1, 2 ] },
+        { "group": "homebooks", "x": 12, "y": [ 12, 14 ], "chance": 70, "repeat": [ 1, 2 ] },
+        { "group": "magazines", "x": 21, "y": [ 15, 17 ], "chance": 30 },
+        { "group": "magazines", "x": 9, "y": 12, "chance": 30 },
+        { "group": "snacks", "x": [ 16, 21 ], "y": 11, "chance": 40, "repeat": [ 1, 2 ] },
+        { "group": "snacks", "x": [ 16, 21 ], "y": 13, "chance": 40, "repeat": [ 1, 2 ] },
+        { "group": "softdrugs", "x": 21, "y": 20, "chance": 75, "repeat": [ 1, 2 ] },
+        { "group": "cleaning", "x": 11, "y": 5, "chance": 70, "repeat": [ 1, 2 ] },
+        { "group": "cleaning", "x": [ 11, 12 ], "y": 8, "chance": 70, "repeat": [ 1, 2 ] },
+        { "group": "surgery", "x": [ 8, 9 ], "y": 9, "chance": 70, "repeat": [ 1, 2 ] },
+        { "group": "bionics_common", "x": 7, "y": 9 },
+        { "item": "anesthetic_kit", "x": 7, "y": 9 },
+        { "item": "television", "x": 21, "y": 16, "chance": 95 },
+        { "item": "soap", "x": 19, "y": 21, "chance": 80 },
+        { "item": "towel", "x": 20, "y": 21, "chance": 80 },
+        { "item": "towel", "x": 3, "y": 13, "chance": 60 },
+        { "item": "stereo", "x": 2, "y": 13, "chance": 50 }
       ],
       "place_monster": [ { "monster": "mon_broken_cyborg", "x": 14, "y": 3, "chance": 100 } ]
     }
   },
   {
     "type": "mapgen",
     "method": "json",
-    "om_terrain": [ "basement" ],
+    "om_terrain": [ "basement_bionic_decoy" ],
```

Comment:
If you give this the same ID, basement_bionic, and just comment on it being a decoy, you won't need separate overmap entries and things. The game will just select which map to use based on weight.

Reply:
I did this bemuse bionic_basement is a scenario start location and I don't want people spawning in the wrong basement model.  I don't do it to other matching basements.

Reply:
Ah, yeah I didn't notice it for the others either. that makes sense.